dask_jobqueue/htcondor.py

```diff
@@ -107,6 +113,9 @@ def __init__(self, disk=None, job_extra=None, config_name="htcondor", **kwargs):
             )
         if self.job_extra:
             self.job_header_dict.update(self.job_extra)
+        if schedd:
+            self.submit_command += " -name "+schedd
+            self.cancel_command += " -name "+schedd
```

Comment:
```suggestion
            self.cancel_command += " -name "+shell_quote(schedd)
```

Reply:
Thanks! I used simply shlex.quote since shlex was already imported as a whole module (see https://github.com/dask/dask-jobqueue/pull/319#issuecomment-523049358) .